ambry-server/src/test/java/com.github.ambry.server/AmbryRequestsTest.java

```diff
@@ -927,6 +1043,97 @@ private void doCatchupStatusTest(PartitionId id, long acceptableLagInBytes, shor
     assertEquals("Catchup status not as expected", expectedIsCaughtUp, adminResponse.isCaughtUp());
   }
 
+  // ttlUpdateTest() helpers
+
+  /**
+   * Does a TTL update and checks for success if {@code expected} is {@link ServerErrorCode#No_Error}. Else, checks for
+   * failure with the code {@code expected}.
+   * @param correlationId the correlation ID to use in the request
+   * @param clientId the client ID to use in the request
+   * @param blobId the blob ID to use in the request
+   * @param expiresAtMs the expiry time (ms) to use in the request
+   * @param opTimeMs the operation time (ms) to use in the request
+   * @throws InterruptedException
```

Comment:
missing `expected` param in java doc. Also, suggest using `expectedErrorCode` or `expectedError`.

Reply:
Done